build.gradle

```diff
@@ -252,6 +252,7 @@ dependencies {
 
   compile group: 'org.apache.tomcat.embed', name: 'tomcat-embed-core', version: '9.0.37'
   compile group: 'org.apache.tomcat.embed', name: 'tomcat-embed-websocket', version: '9.0.37'
+  compile group: 'org.projectlombok', name: 'lombok', version: '1.18.10'
```

Comment:
Why have we added lombok it has not been used. Also we decided not to use lombok on this project.